build.sbt

```diff
@@ -17,29 +17,9 @@ val common = Seq(
   )
 )
 
-lazy val logging = project
-  .settings(common :+ (name := "logging"))
-  .settings(
-    libraryDependencies ++= Seq(
-      "ch.qos.logback" % "logback-classic" % "1.2.3",
-      "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2"
-    )
-  )
-
 lazy val metricsCommon = project
   .in(file("metrics-common")).settings(common :+ (name := "metrics-common"))
 
-
-lazy val kamonMetrics = project
-  .in(file("metrics-kamon"))
-  .settings(common :+ (name := "kamon-metrics"))
-  .dependsOn(metricsCommon)
-  .settings(
```

Comment:
What is exciting about this is you can also use this as a time to clean up the method signature of `Metrics`, which has an extra wrapping into `F[_]` that is pointless

i.e.

```
trait Metrics[F[_]] {
  def counter(metric: Metric): F[Long => F[Unit]]
  def histogram(metric: Metric): F[Long => F[Unit]]
}
```
becomes

```
trait Metrics[F[_]] {
  def counter(metric: Metric)(value: Long): F[Unit]
  def histogram(metric: Metric)(value: Long): F[Unit]
}
```